wasm_bindgen/3rdparty/patches/README.md

```diff
@@ -0,0 +1,16 @@
+# Wasm-bindgen patches
+
+## [resolver](./resolver.patch)
+
+This patch avoids the following issue when updating dependencies
+
+```text
+INFO: Running command line: bazel-bin/wasm_bindgen/3rdparty/crates_vendor.sh '--repin=full'
+
+error: failed to parse manifest at `/tmp/.tmpQivIHn/Cargo.toml`
+
+Caused by:
+  cannot specify `resolver` field in both `[workspace]` and `[package]`
```

Comment:
I wonder if there's a way for us to automatically do this, rather than need people to manually patch Cargo.toml files... I can imagine this may happen for our users too, and avoiding it would be good...

Reply:
I filed https://github.com/bazelbuild/rules_rust/issues/2067